Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -426,6 +427,17 @@ void SetDisc(std::unique_ptr<DiscIO::VolumeDisc> disc,
   bool had_disc = IsDiscInside();
   bool has_disc = static_cast<bool>(disc);
 
+  if (has_disc)
+  {
+    const DiscIO::BlobReader& blob = disc->GetBlobReader();
+    if (!blob.HasFastRandomAccessInBlock() && blob.GetBlockSize() > 0x200000)
```

Comment:
Just curious, what is `0x200000`.  Is it possible to recommend a setting in the "convert feature" to rectify the issue?

Reply:
> Just curious, what is `0x200000`?

2 MiB, the minimum block size of WIA. It seems to be fast enough in most cases.

> Also - is it possible to recommend a setting in the "convert feature" to rectify the issue?

The convert dialog doesn't allow you to pick any block size larger than 2 MiB.